src/App.js

```diff
@@ -4,10 +4,50 @@ import Quiz from './Quiz/Quiz';
 import Button from '@material-ui/core/Button';
 
 const dataQuiz = [
```

Comment:
change the name of this variable dataQuiz for me like data for one quiz not for many.